.github/workflows/test-module.yaml

```diff
@@ -0,0 +1,42 @@
+name: Test
+on: 
+  workflow_dispatch:
+    inputs:
+      kind-version:
+        required: true
+        default: 'kind19'
+      fybrik-version:
+        required: true
+        default: '0.6.0'
+      module-version:
+        required: true
+        default: '0.6.0'
+      module-type:
+        required: true
+        default: 'stream'
+
+env:
+  GO_VERSION: 1.17
+  AWS_PAGER: ""
+  AWS_EC2_METADATA_DISABLED: true
+
+jobs:
+  test-module:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.GO_VERSION }}
+    - name: Install tools
+      run: make install-tools
+    - name: Install AWS cli
+      uses: unfor19/install-aws-cli-action@v1
+      with:
+        version: 2     # default
+        verbose: false # default
+        arch: amd64    # allowed values: amd64, arm64
```

Comment:
I now see ` Warning: Unexpected input(s) 'arch', valid inputs are ['version', 'verbose']` in https://github.com/revit13/data-movement-operator/runs/5021924711?check_suite_focus=true maybe we can try to remove `arch` as the default is `amd64`